### ISSUE TITLE
About pointrend (the way of computing point loss)

### ISSUE THREAD

**Issue by YininKorea** [facebookresearch/detectron2#1339]
How does "pointrend" compute point loss in the part of semantic segmentation? I only saw the part of instance  segmentation (using F.binary_cross_entropy_with_logits).

**Comment by ppwwyyxx** [comment]
Semantic segmentation has not been released.

**Comment by alexander-kirillov** [comment]
We've released the code for semantic segmentation now. Point-wise loss for semantic segmentation is computed here: https://github.com/facebookresearch/detectron2/blob/04958b93e1232935e126c2fd9e6ccd3f57c3a8f3/projects/PointRend/point_rend/semantic_seg.py#L101-L103
